runtime/lua/vim/diagnostic.lua

```diff
@@ -241,6 +244,20 @@ local M = {}
 --- whole line the sign is placed in.
 --- @field linehl? table<vim.diagnostic.Severity,string>
 
+--- @class vim.diagnostic.Opts.Jump
+---
+--- Default value of the {float} parameter of |vim.diagnostic.jump()|.
```

Comment:
To render the default in the docs you could have added.

```suggestion
--- Default value of the {float} parameter of |vim.diagnostic.jump()|.
--- (default: false)
```

Reply:
Thanks, good suggestion. Created a typo PR https://github.com/neovim/neovim/pull/29097